src/components/Navigation/index.js

```diff
@@ -3,27 +3,68 @@ import Navbar from "react-bootstrap/Navbar";
 import Nav from "react-bootstrap/Nav";
 import { NavLink } from "react-router-dom";
 import { useSelector } from "react-redux";
-import { selectToken } from "../../store/user/selectors";
+import { selectToken,selectClient } from "../../store/clients/selectors";
+import { selectBasket } from '../../store/baskets/selectors'
 import NavbarItem from "./NavbarItem";
 import LoggedIn from "./LoggedIn";
 import LoggedOut from "./LoggedOut";
+import './index.css'
+import { FaBeer, FaShoppingCart } from 'react-icons/fa';
+import { IconContext } from "react-icons";
 
 export default function Navigation() {
   const token = useSelector(selectToken);
-
+  const client = useSelector(selectClient)
+  const basket = useSelector(selectBasket)
+  const clientId = client.id
+  console.log('basket is', basket)
+  console.log('basketL' ,basket.length)
+  function counterValue(){
+    if(basket.length === 0){
+      return null
+    } else {
+      return basket.length;
+    }
```

Comment:
this can be done inplace I think and with `&&` instead of if's, like:
```
<h4 className="counter">{basket.length}</h4> // this should work already showing 0, why the need to return null?
```